PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs

```diff
@@ -79,6 +79,14 @@ public static class SnipePokemonTask
                     {
                         foreach (var pokemonLocation in locationsToSnipe)
                         {
+                            var pokeBallsCount = await session.Inventory.GetItemAmountByType(POGOProtos.Inventory.Item.ItemId.ItemPokeBall);
+                            var greatBallsCount = await session.Inventory.GetItemAmountByType(POGOProtos.Inventory.Item.ItemId.ItemGreatBall);
+                            var ultraBallsCount = await session.Inventory.GetItemAmountByType(POGOProtos.Inventory.Item.ItemId.ItemUltraBall);
+                            var masterBallsCount = await session.Inventory.GetItemAmountByType(POGOProtos.Inventory.Item.ItemId.ItemMasterBall);
+
+                            if (pokeBallsCount + greatBallsCount + ultraBallsCount + masterBallsCount < session.LogicSettings.MinPokeballsToSnipe)
+                                return;
+
```

Comment:
Shouldn't this be before the foreach loop so that we can do all the locations and not just 1-2? Or maybe random which location to go to each time?